OpenData/secular printed material - 1921 - 1990/Coraa ny Gael/Coraa ny Gael 1953-08-25/document.csv

```diff
@@ -1,31 +1,31 @@
      Ta’n Turneyr-theayagh Sydney J. Kneale as e ven er ve goaill laaghyn seyrey er ny hellanyn y Keelys Vooar dy Rank.,     The public lawyer Sydney J. Kneale and his wife have taken holidays on the islands of the Great Sound of France (the Channel Islands?),
 "     Ta dooinney goaill laaghyn seyrey ayns Purt Chairn ec y traa t’ayn enmyssit Mark Cregeen. T’eh cheet voish Great Nissenden ayns Buckinghamshire, Sostyn.","     There is a man taking holidays in Port Erin at the current time called Mark Cregeen. He comes from Great Nissenden in Buckinghamshire, England.",
 "Cha row eh rieau er yn ellan ain roie, as cha nel fockle erbee dy Gaelg echey.","He has never been on our island before, and he doesn't have a single word of Manx.",
 "Va mee taggloo rish er y guthan laa jea, as dooyrt eh dy vel eh laccal dy akin fer-ennagh as Gaelg echey.","I was talking to him on the telephone yesterday, and he said he wants to see someone who knows Manx.",
 “Crehon?” dooyrt mish.,“What for?” I said.,
 "“Va Archibald Cregeen (yn focklioar) my yishag vooar tree keayrtyn harrish,” ass eshyn.","“Archibald Cregeen (the dictionary) was my great grandfather three times over,” he said.",
-T’eh er jeet hug yn ellan son shilley beg marish e ven as daa phaitchey as t’eh likal yn ellen [sic]  cho-wheesh nagh vel eh laccal dy gholl erash!,He has come to the isalnd for a little sightseeing with his wife and two children and he likes the island so much that he doesn’t want to go back!,
+T’eh er jeet hug yn ellan son shilley beg marish e ven as daa phaitchey as t’eh likal yn ellen [sic]  cho-wheesh nagh vel eh laccal dy gholl erash!,He has come to the island for a little sightseeing with his wife and two children and he likes the island so much that he doesn’t want to go back!,
 Nagh dooyrt Juan Thobm Kaighin — “Cha jean ad dy bragh goll back”!,Didn’t John Tom Kaighin say — “They will never go back”!,
 "Hooar mee screeuyn voish carrey aym ayns Zealand Noa er y chiaghtyn, as aynjee dinsh eh dooys dy row shenn ven Vanninagh beaghey er yn Ellan Twoaieagh as Gaelg eck ass myghin!  ","     I received a letter from a friend of mine in New Zealand in the week, and in it he told me that there was an old Manx woman living on the North Island and she absolutely knew Manx!",
 "Dooyrt my charrey dy daink eh ny quail ree keayrt ny ghaa, as dy row ee foast er-mayrn as jannoo dy-mie. ","My friend said that he met her many a time, and that she was still alive and doing well. ",
-"Haink y ven shid ass Rhumsaa, as va’n ayr eck far-enmyssit “Johnny the Gull.” V’ad gra dy row ram Gaelg echey-hene neesht.","That woman came from Ramsey, and her father was nicknamed “Johnny the Gull.” They said that he himself knew Manx very well too.",
+"Haink y ven shid ass Rhumsaa, as va’n ayr eck far-enmyssit “Johnny the Gull.” V’ad gra dy row ram Gaelg echey-hene neesht.","That woman came from Ramsey, and her father was nicknamed ‘Johnny the Gull’. They said that he himself knew Manx very well too.",
```

Comment:
Hmm. Is this a UK/US style thing? Other quotes elsewhere and earlier in the sentence are double quotes with punctuation inside, which is US standard.

Reply:
Oh, sorry I see it is nested. My bad, never mind!

Reply:
Okay I have thought about this a bit and may comment further but don't want to overdo it right now and want to thank you for your contribution and encourage you to continue.

Reply:
Yes, I was trying to correct the nesting, but you're quite right that I shouldn't have moved the punctuation outside. As a Brit it's a thing that I absolutely hate, but it's not the right place to fix that here – changing the quotation marks was correct, because they needed to be nested, but I need to grit my teeth and leave the full stops inside the quotation marks even for inside quotes!

So _Johnny the Gull_ does need to be in single quotes, but they can simply replace the double ones.

Reply:
Thanks! I finally realized that the outer quotes are not real quotes per se but string delimiters for the computer so "Johnny the Gull" is not nested after all and can remain double.